languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -19232,6 +19232,46 @@ USA
     <!-- Grammar -->
     <!-- ====================================================================== -->
     <category id="GRAMMAR" name="Grammar" type="grammar">
+        <rule id="SV_AGR_1" name="Subject-verb agreement rule #1" default="temp_off"><!-- experimental -->
+            <antipattern>
+                <token regexp="yes">i|you|we|they</token>
+                <token min="0" postag="RB"/>
+                <token chunk_re="B-VP" postag="VBP?" postag_regexp="yes"/>
+            </antipattern>
+            <antipattern>
+                <token skip="3" postag="SENT_START"/>
+                <token postag="SENT_END"/>
+            </antipattern>
+            <antipattern>
+                <token>there</token>
+                <token>are</token>
+            </antipattern>
+            <pattern>
+                <token postag="SENT_START"/>
+                <token skip="-1" chunk_re=".-NP-singular" max="10">
```

Comment:
The chunker becomes confused if a plural noun is in the middle of a noun cluster. Example:
The **management resources test** in the new proposal show how errors occur.

'management resources test' is identified as a plural noun cluster.